Assets/HoloToolkit/Build/Editor/BuildDeployPortal.cs

```diff
@@ -170,7 +170,7 @@ public static AppDetails QueryAppDetails(string baseAppName, ConnectInfo connect
                 AppList appList = JsonUtility.FromJson<AppList>(appListJSON);
                 for (int i = 0; i < appList.InstalledPackages.Length; ++i)
                 {
-                    string appName = appList.InstalledPackages[i].PackageFullName;
+                    string appName = appList.InstalledPackages[i].Name;
```

Comment:
Ok, thanks! I'll take a look at using the PackageFamilyName for app matching throughout the addon and get a change ready. 

I'd propose we take this change in the meantime, as it makes the internally comparisons consistent - in the current code, if someone doesn't match their package name with their app name, launch will not work. This is something I ran into debugging someone's setup with the addon.